src/main/java/com/synopsys/integration/phonehome/google/analytics/GoogleAnalyticsRequestTransformer.java

```diff
@@ -8,45 +8,72 @@
-        this.trackingId = trackingId;
+        this.apiSecret = apiSecret;
+        this.measurementId = measurementId;
     }
 
     public List<NameValuePair> getParameters() {
-        addParameter(GoogleAnalyticsConstants.API_VERSION_KEY, "1");
-        addParameter(GoogleAnalyticsConstants.HIT_TYPE_KEY, "pageview");
+        addParameter("api_secret", apiSecret);
+        addParameter("measurement_id", measurementId);
+        return parameters;
+    }
 
+    private String getFormattedHitDate() {
+        LocalDate currentDate = LocalDate.now();
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("yyyyMMdd");
+        return currentDate.format(dateTimeFormatter);
+    }
+
+    public JsonObject getPayload() {
         String clientId = generateClientId(phoneHomeRequestBody.getCustomerId(), phoneHomeRequestBody.getHostName());
-        addParameter(GoogleAnalyticsConstants.CLIENT_ID_KEY, clientId);
-        addParameter(GoogleAnalyticsConstants.TRACKING_ID_KEY, trackingId);
-        addParameter(GoogleAnalyticsConstants.DOCUMENT_PATH_KEY, "phone-home");
-
-        // Phone Home Parameters
-        addParameter(GoogleAnalyticsConstants.CUSTOMER_ID, phoneHomeRequestBody.getCustomerId());
-        addParameter(GoogleAnalyticsConstants.HOST_NAME, phoneHomeRequestBody.getHostName());
-        addParameter(GoogleAnalyticsConstants.ARTIFACT_ID, phoneHomeRequestBody.getArtifactId());
-        addParameter(GoogleAnalyticsConstants.ARTIFACT_VERSION, phoneHomeRequestBody.getArtifactVersion());
-        addParameter(GoogleAnalyticsConstants.PRODUCT_ID, phoneHomeRequestBody.getProductName());
-        addParameter(GoogleAnalyticsConstants.PRODUCT_VERSION, phoneHomeRequestBody.getProductVersion());
-        addParameter(GoogleAnalyticsConstants.META_DATA, gson.toJson(phoneHomeRequestBody.getMetaData()));
 
-        return parameters;
+        JsonObject eventParams = new JsonObject();
+        eventParams.addProperty(GoogleAnalyticsConstants.CUSTOMER_ID, phoneHomeRequestBody.getCustomerId());
+        eventParams.addProperty(GoogleAnalyticsConstants.ARTIFACT_ID, phoneHomeRequestBody.getArtifactId());
+        eventParams.addProperty(GoogleAnalyticsConstants.ARTIFACT_VERSION, phoneHomeRequestBody.getArtifactVersion());
+        eventParams.addProperty(GoogleAnalyticsConstants.HOST_NAME, phoneHomeRequestBody.getHostName());
+        eventParams.addProperty(GoogleAnalyticsConstants.META_DATA, gson.toJson(phoneHomeRequestBody.getMetaData()));
+        eventParams.addProperty(GoogleAnalyticsConstants.PRODUCT_ID, phoneHomeRequestBody.getProductName());
+        eventParams.addProperty(GoogleAnalyticsConstants.PRODUCT_VERSION, phoneHomeRequestBody.getProductVersion());
+        eventParams.addProperty(GoogleAnalyticsConstants.HIT_DATE, getFormattedHitDate());
+
+        JsonObject eventObject = new JsonObject();
+        eventObject.addProperty(GoogleAnalyticsConstants.EVENT_NAME_KEY, "bd_hit_from_phone_home_lib_2");
```

Comment:
`bd_hit` is used as the event name in google analytics. so the value must be changed from `bd_hit_from_phone_home_lib_2` to `bd_hit`.

Reply:
changed in https://github.com/blackducksoftware/phone-home-client/pull/17/commits/12563d1323d4be1266feb6f34e10558de2e1f143